aea/test_tools/test_cases.py

```diff
@@ -771,15 +793,15 @@ class AEATestCaseMany(BaseAEATestCase):
     @classmethod
     def setup_class(cls):
         """Set up the test class."""
-        BaseAEATestCase.setup_class()
+        super(AEATestCaseMany, cls).setup_class()
 
     @classmethod
     def teardown_class(cls):
         """Teardown the test class."""
-        BaseAEATestCase.teardown_class()
+        super(AEATestCaseMany, cls).teardown_class()
 
 
-class AEATestCase(BaseAEATestCase):
+class AEATestCase(BaseAEATestCase):  # TODO: does not in use at the moment
```

Comment:
This is used in docs; please remove TODO

Reply:
done